command/ca/certificate.go

```diff
@@ -302,3 +343,69 @@ func (f *certificateFlow) Sign(ctx *cli.Context, token string, csr api.Certifica
 	data := append(pem.EncodeToMemory(serverBlock), pem.EncodeToMemory(caBlock)...)
 	return utils.WriteFile(crtFile, data, 0600)
 }
+
+// CreateSignRequest is a helper function that given an x509 OTT returns a
+// simple but secure sign request as well as the private key used.
+func (f *certificateFlow) CreateSignRequest(token string, sans []string) (*api.SignRequest, crypto.PrivateKey, error) {
+	tok, err := jose.ParseSigned(token)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "error parsing token")
+	}
+	var claims tokenClaims
+	if err := tok.UnsafeClaimsWithoutVerification(&claims); err != nil {
+		return nil, nil, errors.Wrap(err, "error parsing token")
+	}
+
+	pk, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
Any reason we're using ecdsa explicitly instead of using the keys.Generate... which uses the defaults?

Reply:
I copied the code from the CA, and it was like that, I'll change it.